package.json

```diff
@@ -99,7 +99,7 @@
     "crypto-js": "3.1.9-1",
     "dayjs": "1.8.27",
     "ecurve": "1.0.6",
-    "electrum-client": "git+https://github.com/BlueWallet/rn-electrum-client.git",
+    "electrum-client": "git+https://github.com/BlueWallet/rn-electrum-client.git#cc018effafd2256272f348953500c4d27cef0d2f",
```

Comment:
only version from commit cc018effafd2256272f348953500c4d27cef0d2f works properly with these changes?

Reply:
No, thanks. But it would be good to separate GoldWallet from BlueWallet also in this case. I failed.